.github/workflows/ci.yml

```diff
@@ -11,24 +11,23 @@ jobs:
       fail-fast: false
       matrix:
         ruby:
-        - '3.0' # With k8s 1.23
-        - '3.0' # With k8s 1.22
-        - '3.0' # With k8s 1.21
+        # Use unique Ruby versions, or GitHub gets confused when building the rest of the matrix
```

Comment:
Ported from https://github.com/Shopify/krane/pull/860/files, except truncated k8s versions to >=1.19.